### ISSUE TITLE
[CST][ENG] Remove the role="text" from CST claim card headings

### ISSUE THREAD

**Issue by davidakennedy** [department-of-veterans-affairs/va.gov-team#88029]
## Value Statement
**_As a_** Veteran who uses assistive technology
**_I want to_** use interfaces with no non-standard ARIA
**_So that_** my assistive technology works properly

---
### Bug

The claim cards have headings that have a `role="text"` on a `div` inside of each claim card heading. This was used in the past to help control how some screen readers read out this multiline heading, but the ARIA role has been deprecated, and it's best practice not to use it.

For more information, [read this article on role=text](https://www.tpgi.com/using-the-text-role/).

To remediate, remove the `role="text"`.

This isn't a WCAG violation, and not a launch blocker.

1. Go to https://staging.va.gov/track-claims/your-claims
2. Log in with a test user
3. Open up dev tools to view `role="text"` on a `div` inside of each claim card heading

<img width="722" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBeS9NQlE9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--f60449c968ae4d8ee81bfc03d002dc6e51887bdd/Screenshot%202024-07-10%20at%201.02.52%E2%80%AFPM.png" alt="Claims page with heading of IN PROGRESS Claim for compensation Received on June 26, 2024 highlighted with red arrow pointing to role=text in source code." />

## Tasks
- [ ] Remove the `role="text"` from `ClaimCard.jsx`.

## Definition of Done
- [ ] Meets acceptance criteria
- [ ] Passed E2E testing (90% coverage)
- [ ] Passed unit testing (90% coverage)
- [ ] Passed integration testing (if applicable)
- [ ] Code reviewed (internal)
- [ ] Submitted to staging
- [ ] Team approved production verification process
- [ ] Design performs design QA and verifies the implementation matches the design spec
- [ ] Accessibility specialist performs accessibility review (in code or design)
- [ ] Engineering identifies staging users required to test and shares account and credentials with design and product
- [ ] Product performs functional QA and verifies acceptance criteria was met

## Acceptance Criteria
- [ ] The `role="text"` is no longer present on the claim card.

**Comment by pmclaren19** [comment]
Reviewed and added some updates to tasks, questions and ac during eng refinement.
